classes/ActionScheduler_AsyncRequest_QueueRunner.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * ActionScheduler_AsyncRequest_QueueRunner
+ */
+
+defined( 'ABSPATH' ) || exit;
```

Comment:
There's nothing wrong with including this, but I just wanted to note that as long as this file (or any other file) *only* declares a class with no other secondary logic, it's OK to omit this check.